bash-completion/test.json

```diff
@@ -7,7 +7,8 @@
   "type": "bash",
   "cleanup": true,
   "skipWhen": [
-    "vmr-ci" // bash completion script not installed
+    "vmr-ci", // bash completion script not installed
+    "non-packaged",
```

Comment:
Can you add a comment to this line (sort of like what is in the previous line) to indicate why we are skipping this? Same goes for other json files.

Reply:
@omajid, I have added the comment for all the files to specify the reason for skipping the test. Please review the same. thanks..